transform/operations_test.go

```diff
@@ -405,6 +406,33 @@ func TestToCamelCase(t *testing.T) {
 	runOpTests(t, func() transformOperation { return &toCamelCase{} }, tests)
 }
 
+func TestCurrentTime(t *testing.T) {
+	tests := []opTests{
+		{
+			description: "Simple working case",
+			args:        map[string]string{"format": time.RFC3339},
+			want:        time.Now().Format(time.RFC3339),
+		},
+		{
+			description: "Missing arg",
+			args:        map[string]string{},
+			wantInitErr: true,
+		},
+		{
+			description: "Too many args",
+			args:        map[string]string{"format": "RFC3339", "cookies": "failure"},
+			wantInitErr: true,
+		},
+		{
+			description: "Format not predefined",
+			args:        map[string]string{"format": "Mon Jan 2 15:04:05 MST 2006"},
+			want:        time.Now().Format("Mon Jan 2 15:04:05 MST 2006"),
+		},
+	}
+
+	runOpTests(t, func() transformOperation { return &currentTime{} }, tests)
```

Comment:
what was the point of splitting everything out if you never use that special use case for checking if the time is within a certain time limit?